assets/src/components/AssignmentTable.js

```diff
@@ -123,7 +130,7 @@ function AssignmentTable (props) {
   return (
     <RootRef rootRef={tableRef}>
       <TableContainer className={classes.container}>
-        <MTable stickyHeader ref={tableRef}>
```

Comment:
The reason I removed the `stickyHeader` is so that I can add a custom top value instead of default's to `top:0`